.travis.yml

```diff
@@ -0,0 +1,9 @@
+
+language: node_js
+node_js:
+  - "lts/*"
+install:
+    - npm install
+script:
+    - npm run lint
```

Comment:
Do we need this if linting is part of test script?

Reply:
Additionally, if only 'test' is used, it [seems](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Default-Build-Script) like that's the default, so maybe can be omitted